fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -94,7 +94,15 @@
       const url = this.href;
       http('HEAD', url, function(res) {
         if (res.status >= 400 || res.getResponseHeader('Link') == null) {
-          location.href = url;
+          var newLocation = url;
```

Comment:
Would the [fetch API](https://fetch.spec.whatwg.org/) be a possibility?  I know almost nothing about it, but the spec and [this blog post]( https://davidwalsh.name/fetch) suggest that it's possible to use different redirect strategies (e.g. `manual`).  [Browser support](http://caniuse.com/#feat=fetch) is possibly adequate.